src/logic/collectibles/sources/OpenSea.ts

```diff
@@ -84,7 +84,10 @@ class OpenSea {
    * @param {string} network
    * @returns {Promise<{ nftAssets: Map<string, NFTAsset>, nftTokens: Array<NFTToken> }>}
    */
-  async fetchAllUserCollectiblesByCategoryAsync(safeAddress, network) {
+  async fetchAllUserCollectiblesByCategoryAsync(
+    safeAddress: string,
+    network: string,
+  ): Promise<{ nftAssets: unknown; nftTokens: unknown }> {
```

Comment:
why unknown if JSDOC from above contains the correct types?

Reply:
it's better now